arkouda/pdarrayclass.py

```diff
@@ -578,13 +578,13 @@ def max(self) -> Union[np.float64,np.int64]:
 
     def argmin(self) -> np.int64:
         """
-        Return the max of the first minimum value of the array.
+        Return the index of the minimum value of the array.
```

Comment:
This should say the *first* mininum value, i.e. if there are multiple instances of the same minimum value, the lowest of their indices will be returned.

Reply:
updated